packages/registry/src/registry/types.ts

```diff
@@ -6,6 +6,11 @@ export interface IRegistry<AssetMap = any> {
   clear: () => void;
 }
 
+export interface ICompiledComponent {
+  cssPath?: string;
```

Comment:
We're renaming these right?

Reply:
Since I removed it to be under the compiledComponent attribute, I don't think we need to rename it. It is more obvious that this is the css path for the compiled product:
```metaData.compiledComponent.cssPath```

Reply:
Oh yeah, cool 💯 🥇